example/src/examples/http_proxy_repository.dart

```diff
@@ -27,6 +27,9 @@ class HttpProxyRepository extends PackageRepository {
         baseUrl.resolve('/api/packages/${Uri.encodeComponent(package)}');
 
     http.Response response = await client.get(versionUrl);
+
+    if(response.statusCode !=200) return;
```

Comment:
Please be compliant with `dartfmt` and have whitespace between `!=` and `200`..

Just run `dartfmt`, I would recommend using visual studie code with https://dartcode.org/, it's awesome and does automatic code formatting :)

Reply:
Also please be compliant with ["Effective Dart"](https://dart.dev/guides/language/effective-dart/style#do-use-curly-braces-for-all-flow-control-structures) which says: "DO use curly braces for all flow control structures.".

In this case:
```dart
if (response.statusCode != 200) {
  return;
}
```

Reply:
ok, I'll fix it